core/src/main/java/com/linecorp/armeria/common/metric/AbstractMetricCollectingBuilder.java

```diff
@@ -67,7 +70,11 @@ protected MeterIdPrefixFunction meterIdPrefixFunction() {
      *         })
      *         .newDecorator();
      * }</pre>
+     *
+     * @deprecated Use {@link ClientBuilder#successFunction(SuccessFunction)} or
+     *                 {@link ServerBuilder#successFunction(SuccessFunction)} instead.
```

Comment:
'instead` is redundant.
```suggestion
     *                 {@link ServerBuilder#successFunction(SuccessFunction)}.
```